### PR TITLE
[wake-lock] Adapted feature policy tests to latest spec

### PR DESCRIPTION
**Pull request by Honry**

### CODE REVIEW

**Comment by Honry** [comment]
@kenchris, @marcoscaceres, would you PTAL? thanks!

File: wake-lock/wakelock-disabled-by-feature-policy.https.sub.html [hunk 1]
```diff
@@ -11,13 +11,13 @@
   const cross_origin_src =
     "https://{{domains[www]}}:{{ports[https][0]}}" + same_origin_src;
 
-  promise_test(t => {
-    return promise_rejects(t, "SecurityError", navigator.getWakeLock("screen"));
+  test(() => {
+    assert_throws("SecurityError", () => new WakeLock("screen"));
```
Comment (kenchris) [inline wake-lock/wakelock-disabled-by-feature-policy.https.sub.html]: We moved to NotAllowedError as that is more correct

Comment (Honry) [reply wake-lock/wakelock-disabled-by-feature-policy.https.sub.html]: Oh, I may miss the change. Will fix soon.

Comment (Honry) [reply wake-lock/wakelock-disabled-by-feature-policy.https.sub.html]: Done. :)

**Comment by Honry** [comment]
@kenchris, any further comments? Friendly ping. :)

**Review by kenchris — APPROVED** [review]

**Comment by Honry** [comment]
@kenchris, thanks! Merge it as-is.

**Pull request merged**